### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by cmcewen**
### Description

Update dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,21 +44,21 @@
   "devDependencies": {
     "@babel/core": "^7.6.2",
     "@babel/runtime": "^7.6.2",
-    "@types/jest": "^24.0.18",
-    "babel-jest": "^24.9.0",
+    "@types/jest": "^25.1.4",
```
Comment (aaronmgdr) [inline package.json:47]: yay!

Comment (aaronmgdr) [reply package.json:47]: You might need to also upgrade jest-dom in web package to its latest version and change the type on our custom  `isBigNumber` Matcher as the matcher interface for 24 is like `interface Matcher<R>` while 25 has `interface Matcher <R, T>`

File: .circleci/config.yml [hunk 2]
```diff
@@ -99,6 +99,10 @@ commands:
               echo "There are git differences after running yarn install"
               exit 1
             fi
+      - run:
+          name: Fail if there are vulnerabilities in packages
```
Comment (jmrossy) [inline .circleci/config.yml:102]: Nice!

File: .circleci/config.yml [hunk 3]
```diff
@@ -99,6 +99,10 @@ commands:
...
+          command: |
+            /bin/bash -c 'yarn audit --groups dependencies --level high; [[ $? -ge 8 ]] && exit 1 || exit 0'
```
Comment (jmrossy) [inline .circleci/config.yml:104]: Does `yarn audit` work for you? It always fails for me cause our monorepo is too big. Maybe that's been fixed

**Review by jmrossy — APPROVED** [review]
Thanks for taking this on!

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**